bkbit/data_translators/genome_annotation_translator.py

```diff
@@ -670,45 +811,67 @@ def __merge_values(self, t):
         return result
 
     def serialize_to_jsonld(
```

Comment:
perhaps this function could be moved to utils, since likely is needed in other cases